modules/ROOT/pages/import-api-specification-maven.adoc

```diff
@@ -0,0 +1,63 @@
+= Importing an API Specification from Maven
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification in a Maven repository into a new Mule Project, or an existing one.
+
+This feature is possible through API Sync. If you are using an older version of Mule Runtime (4.1.3 and below), or if you don't want to use this new design paradigm and would still prefer to import an API specification that's not in Exchange, follow the xref:studio::import-api-specification-design-center.adoc[Importing an API Specification from a Local File] steps.
```

Comment:
and would still prefer > and want, or > and prefer

Reply:
```suggestion
If you are using Mule runtime engine (Mule) 4.1.3 and earlier, or if you prefer to import an API specification that's not in Exchange, follow the steps presented in xref:studio::import-api-specification-design-center.adoc[Importing an API Specification from a Local File].
```